setup.py

```diff
@@ -24,108 +24,122 @@
 try:
-  import rdkit
+    import rdkit
 except Exception:
-  print('rdkit not found. Attempting to compile rdkit from source ...')
-  with Installer(repo='https://github.com/rdkit/rdkit') as _:
-    pass
-
-if __name__ == '__main__':
-
-  from setuptools import setup, find_packages
-
-  try:
-  	from Cython.Build import cythonize
-  	import numpy
-  	optimal_list = cythonize("src/auto_martini/optimization.py")
-  	include_dirs = [numpy.get_include()]
-  except:
-    print('Failed to cythonize optimization module. For optimal performance, make sure Cython is properly installed.')
-    optimal_list = []
-    include_dirs = []
-
-  setup(
-      name = "auto_martini",
-      version = __version__,
-      author = ["Tristan Bereau", "Andrew Abi-Mansour"],
-      author_email = ["t.bereau [at] uva.nl", "andrew.gaam [at] gmail.com"],
-      description = ("A tool for automatic MARTINI mapping and parametrization of small organic molecules "),
-      license = "GPL v2",
-      keywords = "Coarse-grained Molecular Dynamics, MARTINI Force Field",
-      url = "https://github.com/Andrew-AbiMansour/Auto_MARTINI",
-      packages=find_packages('src'),
-      package_dir={'auto_martini':'src/auto_martini'},
-      package_data={'test': ['test/*.sdf'],},
-      include_package_data=True,
-      install_requires=['numpy', 'bs4', 'pytool', 'lxml', 'requests'],
-      classifiers=[
-          "Development Status :: 2 - Pre-Alpha",
-          "Topic :: Utilities",
-          "License :: OSI Approved :: GNU General Public License v2 (GPLv2)",
-  	     "Programming Language :: Python :: 2.7",
-  	     "Programming Language :: Python :: 3.6"
-      ],
-      zip_safe=False,
-      ext_modules=optimal_list,
-      include_dirs=include_dirs
-  )
+    print("rdkit not found. Attempting to compile rdkit from source ...")
+    with Installer(repo="https://github.com/rdkit/rdkit") as _:
+        pass
+
+if __name__ == "__main__":
+
+    from setuptools import setup, find_packages
+
+    try:
+        from Cython.Build import cythonize
+        import numpy
+
+        optimal_list = cythonize("src/auto_martini/optimization.py")
+        include_dirs = [numpy.get_include()]
+    except:
+        print(
+            "Failed to cythonize optimization module. For optimal performance, make sure Cython is properly installed."
+        )
+        optimal_list = []
+        include_dirs = []
+
+    setup(
+        name="auto_martini",
+        version=versioneer.get_version(),
+        cmdclass=versioneer.get_cmdclass(),
+        author=["Tristan Bereau", "Andrew Abi-Mansour"],
+        author_email=["t.bereau [at] uva.nl", "andrew.gaam [at] gmail.com"],
+        description=(
+            "A tool for automatic MARTINI mapping and parametrization of small organic molecules "
+        ),
+        license="GPL v2",
+        keywords="Coarse-grained Molecular Dynamics, MARTINI Force Field",
+        url="https://github.com/Andrew-AbiMansour/Auto_MARTINI",
```

Comment:
Should we change the URL?
```suggestion
        url="https://github.com/tbereau/auto_martini",
```